pom.xml

```diff
@@ -26,7 +26,8 @@
         <commonsLangVersion>3.11</commonsLangVersion>
         <guavaVersion>30.1-jre</guavaVersion>
         <jacksonVersion>2.12.0</jacksonVersion>
-        <hawtbufVersion>1.1</hawtbufVersion>
+        <activemq-protobufVersion>1.1</activemq-protobufVersion>
+        <hawtbufVersion>1.11</hawtbufVersion>
```

Comment:
is property `hawtbufVersion` used?

Reply:
yes, the same property was used for two dependencies.
` <dependency>
            <groupId>org.fusesource.hawtbuf</groupId>
            <artifactId>hawtbuf</artifactId>
            <version>${hawtbufVersion}</version>
            <scope>provided</scope>
        </dependency>`